### PR TITLE
Unity quest content description

### PR DESCRIPTION
**Pull request by thatrobotdev**
Adds a description for Android build instructions from the explainer image to the article, making the article easier to read (and embedding a link that was initially only embedded in the image). Increases accessibility of the article to screenreaders and other technologies with a long description of what is going on in the image.

### CODE REVIEW

File: Docs/docs/platforms/android/setting-up-unity-for-creating-quest-content.md [hunk 1]
```diff
@@ -6,14 +6,29 @@ createdAt: "2019-04-10T00:53:35.417Z"
...
+For more details on best practices when working with dual-platform projects, read our documentation on [Cross-Platform Setup](/platforms/android/cross-platform-setup).
 
-[You need to install Unity's Android SDK](https://docs.unity3d.com/2019.4/Documentation/Manual/android-sdksetup.html). Otherwise the option won't pop up.
+## 3 Steps to get your world on VRChat for Quest
 
+### 1. Open Your Build Settings
+
+:::tip 
+Pros use the hotkeys Ctrl + Shift + B.
 :::
```
Comment (Faxmashine) [inline Docs/docs/platforms/android/setting-up-unity-for-creating-quest-content.md:21]: Should this contain a link to Unity's documentation?

The shortcut is helpful, but admonitions should only be used for information that's very important.
```suggestion
### 1. Open Your Build Settings

You can access your [Build Settings](https://docs.unity3d.com/Manual/BuildSettings.html) from Unity's main menu by going to "File" -> "Build Settings". Or you can use the keyboard shortcut `Ctrl` + `Shift` + `B`.
```

File: Docs/docs/platforms/android/setting-up-unity-for-creating-quest-content.md [hunk 2]
```diff
@@ -6,14 +6,29 @@ createdAt: "2019-04-10T00:53:35.417Z"
...
\ No newline at end of file
+### 2. Switch Platform to Android
+
+:::caution Requires Additional Setup
+If the Android option isn't appearing, [you'll need to install Unity's Android SDK.](https://docs.unity3d.com/Manual/android-sdksetup.html)
+
+:::
+
+Click "Android" -> "Switch Platform". Unity will automatically convert assets for Android-ready, it will take a while for larger projects with lots of assets.
+
```
Comment (Faxmashine) [inline Docs/docs/platforms/android/setting-up-unity-for-creating-quest-content.md:31]:
```suggestion
### 2. Switch Platform to Android

:::caution Requires Additional Setup
If the Android option isn't appearing, [you need to install Unity's Android SDK.](https://docs.unity3d.com/Manual/android-sdksetup.html)

:::

Click "Android" -> "Switch Platform". Unity will reimport your assets for Android. This takes a while for large projects with many assets.

```

File: Docs/docs/platforms/android/setting-up-unity-for-creating-quest-content.md [hunk 3]
```diff
@@ -6,14 +6,29 @@ createdAt: "2019-04-10T00:53:35.417Z"
...
+### 3. Publish -> New Build
+
+That's it! Your content is ready to check out on the Quest! Note that **you must publish**. Testing locally is not supported for Quest.
```
Comment (Faxmashine) [inline Docs/docs/platforms/android/setting-up-unity-for-creating-quest-content.md:34]: Users need to upload their content, but they don't need to publish it.
```suggestion
That's it! Your content is now available on Quest! Note that you must **upload** your content to test it. You can't use the SDK's "Build and Test" feature for Android content.
```

**Review by Faxmashine — CHANGES_REQUESTED** [review]
Thank you for the pull request! This is definitely an improvement. I left a few additional suggestions.

File: Docs/docs/platforms/android/setting-up-unity-for-creating-quest-content.md (hunk 2 above)
Comment (thatrobotdev) [reply Docs/docs/platforms/android/setting-up-unity-for-creating-quest-content.md]: Added in https://github.com/vrchat-community/creator-docs/pull/166/commits/2b0d4b73b4355c39c4e7b9e9c6aea9df6709d511

**Comment by thatrobotdev** [comment]
Thank you for the changes, I believe I've implemented all of them!

**Review by Faxmashine — APPROVED** [review]

**Pull request merged**